nbdime/diffing/notebooks.py

```diff
@@ -194,6 +195,8 @@ def diff_mime_bundle(a, b, path=None):
     "/cells/*": diff,
     "/cells/*/outputs": diff_sequence_multilevel,
     "/cells/*/outputs/*": diff_single_outputs,
+    "/cells/*/attachments": diff_sequence_multilevel,
+    "/cells/*/attachments/*": diff_mime_bundle,
```

Comment:
(or more generally a `move` op, but a move op for arrays is likely to add more complexity)